### PR TITLE
added terraform for redash

### PR DESCRIPTION
**Pull request by omry-hay**

### CODE REVIEW

File: redash/aws/ec2.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+resource "aws_instance" "redash-instance" {
+  ami = "ami-0c654c3ab463d22f6"
+  instance_type = "t2.small"
```
Comment (onhate) [inline redash/aws/ec2.tf]: will the user be able somehow to chose instance types?

**Review by onhate — COMMENTED** [review]
as a general comment I'm thinking if we will support multiple versions of redash (or AMIs for different regions) maybe the directory structure should be `redash/<version>/<identifier>/` where `<identifider>` in this case will be AMI.

**Comment by ohad1282** [comment]
Looks great. Our first GitHub TF files that are used as templates! For sure should be more configurable and with control to the user in the future (like instance type/size, maybe not even coupled to the static AMIs of redash (as they change every now and then) and rather build the vm more dynamically). For now it is a great start IMHO.

File: redash/aws/ec2.tf (hunk 1 above)
Comment (omry-hay) [reply redash/aws/ec2.tf]: Not sure that it will be available for the MVP, but it would be probably in the next phases.

**Comment by omry-hay** [comment]
@onhate - You are right, I will try to come up with a better way to do that.
However it is good enough to start working on it, and it is a good way to develop and debug the deployment service.

**Review by onhate — APPROVED** [review]

**Pull request merged**